pom.xml

```diff
@@ -25,7 +25,7 @@
         <maven.compiler.target>${jdkVersion}</maven.compiler.target>
 
         <!-- Versioning -->
-        <version.number>1.12.3</version.number>
+        <version.number>1.12.4</version.number>
```

Comment:
"I normally bump the version only when I know that I won't commit something anymore."

Really strange to do it like this ^^
Normally you create a branch with the new version on which you are working the whole time, while the master stays on the last stable... or not?